src/features/contactmoment/ContactmomentVragenMenu.vue

```diff
@@ -52,6 +52,9 @@ async function startNieuweVraag() {
     await waitForConfirmation();
   }
   contactmomentStore.startNieuweVraag();
+  setTimeout(() => {
+    document.getElementById("cm-notitieblok")?.focus();
+  });
```

Comment:
(This might be the wrong Vue version, but something like:)

```
mounted: function () {
  this.$nextTick(function () {
    // Code that will run only after the
    // entire view has been rendered
  })
}
```

Reply:
The `setTimeout` default second param is set to `0`, basically making this timeout not "timing out". It does seem to work though.